tests/robot-tests/tests/libs/public-api-common.robot

```diff
@@ -46,3 +46,22 @@ verify status of API Datasets
     user waits for caches to expire
     ${status_value}=    get text    xpath:(//div[@data-testid="Status"]//dd[@data-testid="Status-value"]//strong)[2]
     should be equal as strings    ${status_value}    ${expected_status}
+    
+user checks value inside the column heading in the Draft version table
```

Comment:
If this is checking the status specifically, I think this can be renamed to something like `user checks status in Draft version table`?